aspnetcore/signalr/groups.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Manage users and groups in SignalR
+author: 
+description: 
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: 
+ms.custom: mvc
+ms.date: 06/04/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/groups
+---
+
+# Manage users and groups in SignalR
+
+## Users?
+
+A user is one or more connections associated with an identifier. By default SignalR will use the `ClaimTypes.NameIdentifier` from the `ClaimsPrincipal` associated with the connection. There can be multiple connections for a single user. For example you can be connected on your desktop, your phone, and your laptop. If you send to the user all three connections will receive the message.
```

Comment:
I would avoid treating a "user" as a concrete object here. So instead of "A user is one or more connections..." something like "SignalR allows you to send messages to all connections associated with a specific User. By default, SignalR will use the (blah blah claims) associated with the connection as the user identifier. Multiple connections can be associated with the same user. For example, ..."